.gitlab-ci.yml

```diff
@@ -0,0 +1,23 @@
+image: alpine:3.9
+
+variables:
+  SAUCECTL_VERSION: 0.44.0
+  SAUCE_USERNAME: ${SAUCE_USERNAME}
+  SAUCE_ACCESS_KEY: ${SAUCE_ACCESS_KEY}
+
+stages:
+  - test
+
+.saucectl:
+  before_script:
+    - apk add curl
+    - curl -L -o saucectl.tar.gz https://github.com/saucelabs/saucectl/releases/download/v${SAUCECTL_VERSION}/saucectl_${SAUCECTL_VERSION}_linux_64-bit.tar.gz
+    - tar -xvzf saucectl.tar.gz
+    - mv saucectl /usr/bin/saucectl
+
+test sauce:
+  extends: .saucectl
+  image: docker:${DOCKER_VERSION}
+  stage: test
+  script:
+    - saucectl run -c .sauce/config.yml --test-env sauce --ccy 10
```

Comment:
`--test-env` has been deprecated for many weeks now. We should stop including it in our examples ⚠️ 

```suggestion
    - saucectl run -c .sauce/config.yml --ccy 10
```

Reply:
Thanks! I fixed this issue.